src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebGitRouteConfig.kt

```diff
@@ -1,15 +1,33 @@
 package org.vaccineimpact.orderlyweb.app_start.routing.web
 
 import org.vaccineimpact.orderlyweb.*
+import org.vaccineimpact.orderlyweb.EndpointDefinition
 import org.vaccineimpact.orderlyweb.app_start.RouteConfig
 import org.vaccineimpact.orderlyweb.controllers.web.GitController
+import org.vaccineimpact.orderlyweb.json
+import spark.route.HttpMethod
 
 object WebGitRouteConfig : RouteConfig
 {
     private val runReports = setOf("*/reports.run")
     override val endpoints: List<EndpointDefinition> = listOf(
             WebEndpoint("/git/branch/:branch/commits/",
                    GitController::class, "getCommits")
+                    .secure(runReports)
+                    .json(),
+
+            WebEndpoint("/git/status/",
```

Comment:
I thought I may as well add all the endpoints while I'm here

Reply:
Don't think we need status and pull do we?